geode-core/build.gradle

```diff
@@ -104,6 +104,8 @@ task createVersionPropertiesFile(dependsOn: ':writeBuildInfo') {
   }
 }
 
+ext.moduleName = group + '.core'
```

Comment:
I'm really not a fan of bloating out the `ext` in our project / subprojects.  Is there a reason this shouldn't just be in-lined below?

Reply:
The modules "plugin" expects it project. It is consistent with the experimental plugin from Gradle. Eventually they will have first level support for this.

Reply:
Should also mention that it goes away completely with the upcoming `module-info.java` changes.

Reply:
Ah, makes sense and good to hear.